.circleci/config.yml

```diff
@@ -1304,15 +1304,6 @@ workflows:
            - CI_MEASURE
   gcp-ehp:
     jobs:
-      - build-and-push-image:
-          filters:
-            tags:
-              only: /^v.*/
-            branches:
-              ignore: /.*/
-          context:
-            - GCP
-            - GCP_EHP
```

Comment:
De ce que je comprends avant on avait 2 workflows en un  : 
- tag v* où on faisait uniquement build-and-push-image
- branchs staging integration où on faisait tout le reste
Dans les deux cas le workflow était gcp-ehp c'est ça ?

Et maintenant il se lancera uniquement dans le deuxième cas ?

Reply:
C'est bien ça !